plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -355,8 +357,10 @@ public void dropTable(ConnectorSession session, SchemaTableName schemaTableName)
         catch (AmazonServiceException e) {
             throw new TrinoException(HIVE_METASTORE_ERROR, e);
         }
-        dropTableData(table.io(), table.operations().current());
```

Comment:
Would it be possible to make `table.io()` return an instance of `SupportsBulkOperations`
so that we don't need to remember to wrap it in every place where it matters?